### ISSUE TITLE
Building bin/qbg is happening even with NGT_QBG_DISABLED

### ISSUE THREAD

**Issue by lerouxrgd** [yahoojapan/NGT#157]
Hello,

I noticed that even though I set `NGT_QBG_DISABLED=ON`  CMake will build `bin/qbg` due to:

```cmake
add_subdirectory("${PROJECT_SOURCE_DIR}/bin/qbg")
```

This actually prevents `ngt-rs` from building without QBG (testing with `cargo test`) when using NGT `2.2.1`.
You can actually test it locally on [this branch](https://github.com/lerouxrgd/ngt-rs/tree/ngt-2.2).

I guess the cause is the following `if` statement:

```cmake
	if(NOT DEFINED NGT_SHARED_MEMORY_ALLOCATOR OR (NOT ${NGT_SHARED_MEMORY_ALLOCATOR}))
		add_subdirectory("${PROJECT_SOURCE_DIR}/bin/qbg")
        endif()
``` 

Would it be possible to check `NGT_QBG_DISABLED` instead of `NGT_SHARED_MEMORY_ALLOCATOR` in this if statement ?

**Comment by masajiro** [comment]
Hello,
Thank you for your helpful comment to solve this issue. I have released [V2.2.2](https://github.com/yahoojapan/NGT/releases/tag/v2.2.2) for this issue.

**Comment by lerouxrgd** [comment]
Thank you for the fix ! I can build it correctly now !
